storage/client/blob/src/main/java/com/azure/storage/blob/models/package-info.java

```diff
@@ -3,6 +3,6 @@
 // Code generated by Microsoft (R) AutoRest Code Generator.
 
 /**
- * This package contains the data models for AzureBlobStorage.
+ * Package containing the data models for AzureBlobStorage.
```

Comment:
This was generated by codegen, this will be overwritten next time it is generated

Reply:
Is there a way to update the codegen to use `Package containing` prefix instead? It makes it consistent with the rest of the package description here - https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/apidocs/index.html